packages/core/src/render3/util/styling_utils.ts

```diff
@@ -455,3 +449,192 @@ export function splitOnWhitespace(text: string): string[]|null {
 export function selectClassBasedInputName(inputs: PropertyAliases): string {
   return inputs.hasOwnProperty('class') ? 'class' : 'className';
 }
+
+/**
+ * Returns a style entry with the `prop` and `value` concatenated together
+ */
+export function concatStyle(prop: string, value: string) {
+  return `${prop}${STYLE_PROP_VALUE_SEPARATOR}${value}`;
```

Comment:
```suggestion
  return prop +STYLE_PROP_VALUE_SEPARATOR + value;
```
Is a lot more efficient